index.html

```diff
@@ -74,6 +74,10 @@
 
   <script src="bower_components/firebase/firebase.js"></script>
   <script>
+
+    window.ENV = 'dev';
```

Comment:
do you want to have like this in config?
"environment": {
    "dev": "dev",
    "prod": "prod"
  },